MicroServices/ShopFlowPro-service/src/main/java/com/wsu/shopflowproservice/service/ServiceService.java

```diff
@@ -0,0 +1,39 @@
+package com.wsu.workorderproservice.service;
+
+import com.wsu.workorderproservice.repository.ServiceRepository;
+import com.wsu.workorderproservice.dto.ServiceDTO;
+import com.wsu.workorderproservice.model.Service;
+import com.wsu.workorderproservice.exception.DatabaseErrorException;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.cglib.core.CollectionUtils;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@RequiredArgsConstructor
+public class ServiceService {
+
+    private final ServiceRepository serviceRepository;
+
+    public List<ServiceDTO> get(boolean isActive) {
+        try {
+            if (isActive) {
+                return mapToDTO(serviceRepository.findByActive(true));
```

Comment:
you don't have active status in entity model class and table. You can safely delete this method call and remove this method from repository as well. 

You can have findAll method only to return all service types.